doc/api/Gateway.md

```diff
@@ -0,0 +1,163 @@
+Gateway API
+===========
+
+This document contains detailed descriptions of the gateway's API routes. For
+an overview of the gateway's API routes, see
+[API.md#gateway](/doc/API.md#gateway).  For an overview of all API routes, see
+[API.md](/doc/API.md)
+
+There may be functional API calls which are not documented. These are not
+guaranteed to be supported beyond the current release, and should not be used
+in production.
+
+Overview
+--------
+
+The gateway maintains a peer to peer connection to the network and provides a
+method for calling RPCs on connected peers. The gateway's API endpoints expose
+methods for viewing the connected peers, manually connecting to peers, and
+manually disconnecting from peers. The gateway may connect or disconnect from
+peers on its own.
+
+Index
+-----
+
+| Route | HTTP verb |
+| ----- | --------- |
+| [/gateway](#gateway-get) | GET |
+| [/gateway/connect/{netaddress}](#gatewayconnectnetaddress-post) | POST |
+| [/gateway/disconnect/{netaddress}](#gatewaydisconnectnetaddress-post) | POST |
+
+Examples
+--------
+
+* [gateway info](#gateway-info-example)
+* [connecting to a peer](#connect-example)
+* [disconnecting from a peer](#disconnect-example)
+
+-------------------------------------------------------------------------------
+
+#### /gateway [GET]
+
+`/gateway [GET]` returns information about the gateway, including the list of
+connected peers.
+
+###### JSON Response
+```javascript
+{
+    // netaddress is the network address of the gateway as seen by the rest of
+    // the network. The address consists of the external IP address and the
+    // port Sia is listening on. It represents a `modules.NetAddress`.
+    "netaddress": String,
+
+    // peers is an array of peers the gateway is connected to. It represents
+    // an array of `modules.Peer`s.
+    "peers":      []{
+        // netaddress is the address of the peer. It represents a
+        // `modules.NetAddress`.
+        "netaddress": String,
+
+        // version is the version number of the peer.
+        "version":    String,
+
+        // inbound is true when the peer initiated the connection. This field
+        // is exposed as outbound peers are generally trusted more than inbound
+        // peers, as inbound peers are easily manipulated by an adversary.
+        "inbound":    Boolean
+    }
+}
+```
+
+###### [Example](#gateway-info-example)
```

Comment:
line 1: bold
line 2:  normal, with `code` at the front
line 3: faded bold
line 4-12: block
line 13: link
line 14: bold
line 15-16: normal, leading `with code`
line 17: faded bold
line 18 `code`
line 19: faded bold
line 20: normal
line 21: link

In 21 lines you've got 12 format changes. That's really difficult for me to read. 

From what I can tell, each route currently has a maximum of 7 different style changes:

route
route description
parameters title
parameters javascript
response title
response javascript / text
example

Ideally we could get this down to 3 or 4. I'm not sure we can get that low, but maybe you've got ideas. Perhaps:

route + example link
description
parameters javascript
response javascript